commands/cp.py

```diff
@@ -11,5 +12,4 @@ def execute(self, cls):
             destination = data[1]
         except:
             print "Invalid syntax of cp command"
```

Comment:
can you please change print "Invalid syntax of cp command" to print("Invalid syntax of cp command")
-> print statement should be same in all the files.